packages/compass-indexes/src/modules/index.ts

```diff
@@ -38,21 +24,16 @@ const reducer = combineReducers({
 });
 
 export type RootState = ReturnType<typeof reducer>;
+export type IndexesThunkDispatch<A extends AnyAction> = ThunkDispatch<
+  RootState,
+  unknown,
+  A
+>;
+export type IndexesThunkAction<R, A extends Action = AnyAction> = ThunkAction<
+  R,
+  RootState,
+  unknown,
+  A
+>;
 
-const rootReducer = (state: RootState, action: AnyAction): RootState => {
-  if (action.type === RESET_FORM) {
-    return {
-      ...state,
-      isWritable: WRITABLE_INITIAL_STATE,
-      isReadonlyView: READONLY_VIEW_INITIAL_STATE,
-      description: DESCRIPTION_INITIAL_STATE,
-      serverVersion: SV_INITIAL_STATE,
-      namespace: NAMESPACE_INITIAL_STATE,
-      regularIndexes: REGULAR_INDEXES_INITIAL_STATE,
-      searchIndexes: SEARCH_INDEXES_INITIAL_STATE,
-    };
-  }
-  return reducer(state, action);
-};
```

Comment:
Removed this as we do not have any form fields in view-indexes. This clean up is only applicable to create/drop indexes

Reply:
Awesome, those RESET actions are the biggest mess in old reducers in Compass, nice that we can remove it so easily